src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -152,8 +169,13 @@ public Response startMonitor(
         session.save(sessionModel);
         GTFSDB.commitAndCloseSession(session);
 
-        //Extract the Url and gtfsId to start the background process
-        startBackgroundTask(gtfsRtFeed, updateInterval);
+        if(feedSession == null) {
```

Comment:
@barbeau @kevinsafon Please don't mind the phrasing of words.
If the below scenario is part of the issue #215, i.e.,
>>If a monitoring session for a given feed has been started within 15 min, the code wont create a new task and will just keep the previous one running.

This might be a problem when a user starts monitoring a feed and closes it, then again starts the same feed within 15 minutes of closing the last one, the application does not monitor the latest feed as it gets `feedSession` value not null.

@barbeau Could you please comment on this.